test-runner-poc/environments.js

```diff
@@ -15,9 +15,9 @@ class EnvironmentBase {
   setup(callback) {
     callback();
   }
-  constructor(parsedURI) {
-    this.host = parsedURI.hosts[0].host;
-    this.port = parsedURI.hosts[0].port;
+  constructor(status) {
+    this.port = parseInt(status.primary.split(':')[1]);
```

Comment:
What if this is called with a standalone? Won't `status.primary` not exist?

Reply:
fixed with an if statement